pkgs/development/libraries/kde-frameworks/extra-cmake-modules/setup-hook.sh

```diff
@@ -59,19 +59,15 @@ xdgDataSubdirs=( \
     "wallpapers" "applications" "desktop-directories" "mime" "appdata" "dbus-1" \
 )
 
-ecmHostPathSeen=( )
+declare -gA ecmHostPath
 
 ecmUnseenHostPath() {
```

Comment:
how about ecmEnsureUnseenHostPath ?
It's a bit of a mouthful but I feel it describes better the situation.
Again it's a matter of style, so feel free to ignore if not to your liking.